backend/test/test-run.sh

```
@@ -0,0 +1,63 @@
+# Create AWS Secret
+oc -n $NAMESPACE create secret generic $SECRET_NAME --from-literal=AWS_ACCESS_KEY=$AWS_ACCESS_KEY_ID --from-literal=AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
+
+# Create DSPA deployment
+export DEPLOYMENT_NAME="ds-pipeline-$DSPA_NAME"
+oc apply -n $NAMESPACE -f "$temp_file"
+timeout 1m bash -c "until oc -n $NAMESPACE get deployment $DEPLOYMENT_NAME &> /dev/null; do echo 'Waiting for the deployment $DEPLOYMENT_NAME...'; sleep 10; done"
+oc wait --for=condition=available deployment/ds-pipeline-$DSPA_NAME --timeout=10m -n $NAMESPACE
+
+# Get API URL
+echo "Fetching route to $DEPLOYMENT_NAME"
+timeout 1m bash -c "until oc -n $NAMESPACE get route "$DEPLOYMENT_NAME" &> /dev/null; do echo 'Waiting for the route to $DEPLOYMENT_NAME...'; sleep 10; done"
```

Shellcheck (0.11.0)

[warning] The surrounding quotes actually unquote this. Remove or escape them.

(SC2027)